scr/main/java/bittercode/util/StoreUtil.java

```diff
@@ -0,0 +1,86 @@
+package com.bittercode.util;
+
+import java.io.PrintWriter;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpSession;
+
+import com.bittercode.model.UserRole;
+
+/*
+ * Store UTil File To Store Commonly used methods
+ */
+public class StoreUtil {
+
+    /**
+     * Check if the User is logged in with the requested role
+     */
+    public static boolean isLoggedIn(UserRole role, HttpSession session) {
+
+        return session.getAttribute(role.toString()) != null;
+    }
+
+    /**
+     * Modify the active tab in the page menu bar
+     */
+    public static void setActiveTab(PrintWriter pw, String activeTab) {
+
+        pw.println("<script>document.getElementById(activeTab).classList.remove(\"active\");activeTab=" + activeTab
+                + "</script>");
+        pw.println("<script>document.getElementById('" + activeTab + "').classList.add(\"active\");</script>");
+
+    }
+
+    /**
+     * Add/Remove/Update Item in the cart using the session
+     */
+    public static void updateCartItems(HttpServletRequest req) {
+        String selectedBookId = req.getParameter("selectedBookId");
+        HttpSession session = req.getSession();
+        if (selectedBookId != null) { // add item to the cart
+
+            // Items will contain comma separated bookIds that needs to be added in the cart
+            String items = (String) session.getAttribute("items");
+            if (req.getParameter("addToCart") != null) { // add to cart
+                if (items == null || items.length() == 0)
+                    items = selectedBookId;
+                else if (!items.contains(selectedBookId))
+                    items = items + "," + selectedBookId; // if items already contains bookId, don't add it
+
+                // set the items in the session to be used later
+                session.setAttribute("items", item);
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider fixing variable name typo</b></div>
<div id="fix">

There appears to be a typo in the variable name. The code uses `item` instead of `items` when setting the session attribute.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<blockquote>Check the AI-generated fix before applying</blockquote>
<div id="code">

```
 @@ -51,1 +51,1 @@
 -                session.setAttribute("items", item);
 +                session.setAttribute("items", items);
```


```suggestion
                session.setAttribute("items", items);
```



</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/PrajaktaBendreBito/BookStore/pull/7#issuecomment-2527741727>#f9242c</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged